src/main/java/org/folio/circulation/domain/LoanRenewalService.java

```diff
@@ -27,7 +27,7 @@ private LoanRenewalService(LoanPolicyRepository loanPolicyRepository) {
   }
 
   public CompletableFuture<HttpResult<Loan>> overrideRenewal(Loan loan, DateTime dueDate, String comment) {
-    return loanPolicyRepository.lookupLoanPolicy(loan)
+    return loanPolicyRepository.lookupLoanPolicy(loan, DateTimeZone.UTC)
```

Comment:
Why is UTC hardcoded here, is it because timezone is not yet supported for overridden renewal?

Reply:
yes, it is not yet supported for overridden renewal.
In the current implementation, we have provided the ability to change the time zone.
Perhaps in the future, this will be taken into account.

Reply:
Is the scope for supporting timezone in renewal overrides in this story, or a different story, as part of this feature?

If you don't know, that's ok, I will ask on the story in JIRA.

Reply:
Revert changes